client-app/shared/layout/components/search-bar/search-bar.vue

```diff
@@ -1,99 +1,86 @@
 <template>
-  <div
-    class="flex items-center justify-between px-12 py-7 bg-[color:var(--color-search-bar-bg)] select-none"
-    v-click-outside="() => searchDropdownVisible && hideSearchDropdown()"
-  >
-    <router-link to="/">
-      <VcImage :src="$cfg.logo_inverted_image" class="h-12 mr-8" />
```

Comment:
`VcImage` isn't used anywhere in the file, however it still present in imports, causing warning. Please, remove VcImage import from script tag.